spec/migrations/20160115111829_chargeback_rate_detail_currency_not_nil_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "spec_helper"
+require_migration
+
+describe ChargebackRateDetailCurrencyNotNil do
+  migration_context :up do
+    before(:each) do
+      @cbd = FactoryGirl.create(:chargeback_rate_detail,
```

Comment:
Here's where you'd use the migration_stub (the stub of the class you put in the migration) to create the rows and do your testing of the migration.  see [here](https://github.com/ManageIQ/manageiq/blob/beb1993e3fd92663869e75c0d1ad3543bb9487b2/spec/migrations/20141219222843_remove_miq_worker_rows_without_model_spec.rb#L5)